### PR TITLE
Personal names of additions in Tanasee1

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created very many new person IDs and updated some existing IDs for persons I found in the additions and extras in Tanasee1. The more difficult cases have been already discussed in #2410, #2421, #2429, #2434 and #2434.

### CODE REVIEW

**Review by thea-m — CHANGES_REQUESTED** [review]
Oh wow :D Maybe in the future, rather several smaller PRs might make the review process both humanly and informatically easier to manage.
I have not read the files yet. Quickly skimming I noticed that the app once again introduced the prefix "t:" and the attribute and value xmlns:t="http://www.tei-c.org/ns/1.0" to langUsage and person (maybe also elsewhere, I did not check all files) in some of the records. I believe this has to be corrected (=deleted) manually. Could you do it?

**Comment by CarstenHoffmannMarburg** [comment]
I did not get it. I checked PRS14162WaldaGirgis, PRS14169KrestosMadhen and PRS14148Nob and did not see an element  t: or xmlns:t="http://www.tei-c.org/ns/1.0 in before person in listPerson or in before language in langUsage. It perhaps happened just in a few records. Can you give an example to me to be sure, that I have a view on the right spot?

**Comment by CarstenHoffmannMarburg** [comment]
I see it in PRS14167MaataGwane. It happened not in every but just in a few records.

To be clear: You want me to remove xmlns:t="http://www.tei-c.org/ns/1.0" and t: in every langUsage description. In the element person, I did not come across so far.

**Comment by thea-m** [comment]
Exactly, it happened randomly (?) in some of the records (as far as I know, we still don't understand why this happens and cannot control it...)
For example, in PRS14048Petros, there is
 `  <t:title xmlns:t="http://www.tei-c.org/ns/1.0">ጴጥሮስ</t:title>` 
` <t:profileDesc xmlns:t="http://www.tei-c.org/ns/1.0"> `(And all child elements)
  `<t:revisionDesc xmlns:t="http://www.tei-c.org/ns/1.0">` (And all child elements)
  `<t:text xmlns:t="http://www.tei-c.org/ns/1.0">`
 ` <t:listPerson>`
the concerned elements should be corrected to `<title>...</title>` etc.

File: new/PRS14119GalagayEste.xml
Comment (thea-m) [inline new/PRS14119GalagayEste.xml:49]: I think the name is rather ʾƎšǝte (but could not find it in the dictionary right now)

File: new/PRS14123TafalaGT.xml
Comment (thea-m) [inline new/PRS14123TafalaGT.xml:49]: Tāfalla?

Comment (thea-m) [reply new/PRS14123TafalaGT.xml:49]: Žammara?

File: new/PRS14124WebAyyahu.xml
Comment (thea-m) [inline new/PRS14124WebAyyahu.xml:50]: dāññā?

File: new/PRS14131LabetoGasso.xml
Comment (thea-m) [inline new/PRS14131LabetoGasso.xml:49]: does the first part maybe refer to the title ʾabeto?

File: new/PRS14151Zaiyasus.xml
Comment (thea-m) [inline new/PRS14151Zaiyasus.xml:49]: Zaʾiyasus

File: new/PRS14152AhabHawaryat.xml
Comment (thea-m) [inline new/PRS14152AhabHawaryat.xml:48]: transliteration

File: new/PRS14158Bahaylamaryam.xml
Comment (thea-m) [inline new/PRS14158Bahaylamaryam.xml:49]: Baḫāylamāryām

File: new/PRS14159TanseaKrestos.xml
Comment (thea-m) [inline new/PRS14159TanseaKrestos.xml:57]: reference to entitites external to this recod should be made as `<ref type="mss" corresp="Tanasee1"/>`

File: new/PRS14163AbetoSalabaster.xml
Comment (thea-m) [inline new/PRS14163AbetoSalabaster.xml:49]: I think Abeto  is a title

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! My comments are kept very short due to time constraints, not meant to be impolite :)
I also had difficulties with several of the names and titles. Maybe others will have more suggestions.

File: PRS8001-9000/PRS8595SayfaAr.xml
Comment (CarstenHoffmannMarburg) [reply PRS8001-9000/PRS8595SayfaAr.xml:59]: Alright, corrected it to ሰይፈ፡ ዐርአድ፡ and found in addition ሰይፈ፡ አርዓድ፡

File: PRS10001-11000/PRS10342Yeshaq.xml
Comment (CarstenHoffmannMarburg) [reply PRS10001-11000/PRS10342Yeshaq.xml:60]: Yes, I checked the text again and in comparison to others it looks more like a እ rather than a አ. There is not need to encode an odd orthography.

File: new/PRS14048Petros.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14048Petros.xml:51]: So, my suggestion is to change it to transliteration as it is preferred for place names.

File: new/PRS14059MariGeta.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14059MariGeta.xml:49]: Done.

File: new/PRS14060WalattaIyasus.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14060WalattaIyasus.xml:49]: Done: Walatta ʾIyāsus

File: new/PRS14067HabtuSarad.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14067HabtuSarad.xml:8]: I do not know, whether the second part belongs to the name of this person or whether it is a title for the following person or has another meaning. If there is no better understanding, I will skip these two words from the title as well.

File: new/PRS14071HabtaMG.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14071HabtaMG.xml:49]: As far as I understand, gərā means 'left' in Amharic. If it is not part of the name, it may be seen as part of the military title bətodad e.g. bətodad gərā = "commander of the left (wing)" or "general (to be seated) to the left (of the ruler/commander in chief)". Is it a possible solution to add gərā to the occupation bətodad, although it is split up and the name in between ብቶደደ፡ ሐብተ፡ ማርያም፡ ግራ፡)? Or is it more useful to see it as part of the nomenclature of the following person:  ግራ፡ ብቶደደ፡ ናዜ፡

File: new/PRS14074Gersam.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14074Gersam.xml:49]: I read it as 'Gerśām'
Screenshot 2023-10-24_Gersam

File: new/PRS14088TaklaSyon.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14088TaklaSyon.xml:51]: Done.

File: new/PRS14088TaklaSyon.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14088TaklaSyon.xml:54]: Ok. I will delete the reference to the exact addition and extra.

File: new/PRS14092EraqMasera.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14092EraqMasera.xml:50]: Done as before in PRS14076GabraNazrawi.

File: new/PRS14105Damatewos.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14105Damatewos.xml:49]: Done.

File: new/PRS14106ZaraAbreham.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14106ZaraAbreham.xml:52]: Done.

File: new/PRS14109Mertenehah.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14109Mertenehah.xml:48]: I agree.

File: new/PRS14116Tawaqa.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14116Tawaqa.xml:49]: Done.

File: new/PRS14119GalagayEste.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14119GalagayEste.xml:49]: No problem with changing it.

File: new/PRS14123TafalaGT.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14123TafalaGT.xml:49]: Done.

File: new/PRS14124WebAyyahu.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14124WebAyyahu.xml:50]: Yes, done.

File: new/PRS14131LabetoGasso.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14131LabetoGasso.xml:49]: Indeed I have severe difficulties to understand the text. As far as I understood, የገዛቸውን፡ means 'she was bought by him'. If it is ʾabeto Gāššo, he must be the object (= logic subject) of the sentence and we need another female person, who was bought. Also I find the character has little similarity with an አ, although this may happen in very quick handwriting.
Screenshot 2023-10-24_Labeto

File: new/PRS14152AhabHawaryat.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14152AhabHawaryat.xml:48]: Done.

File: new/PRS14158Bahaylamaryam.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14158Bahaylamaryam.xml:49]: Done.

File: new/PRS14159TanseaKrestos.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14159TanseaKrestos.xml:57]: I changed to the phrase, that you suggested and added the folio numbers. Without pointing to the exact addition (a12), the sentence would be otherwise incomprehensive.

File: new/PRS14163AbetoSalabaster.xml
Comment (CarstenHoffmannMarburg) [reply new/PRS14163AbetoSalabaster.xml:49]: Done.

**Review by thea-m — APPROVED** [review]
Thank you! I would only change Tāwaqa to Tāwwaqa (sorry, didn't notice it before), otherwise it can be merged from my perspective

**Comment by CarstenHoffmannMarburg** [comment]
Changed to Tāwwaqa.

**Comment by eu-genia** [comment]
This is all very nice, thank you, but I am afraid our current gitsync won't work with 113 files; it cannot manage more than 10-15 files at a time. It would be great if this PR can be split in manageable PRs - this also makes it easier for the reviewers. It is quite impossible to produce a review of more than 10 records at a time. 
Thank you!

**Comment by eu-genia** [comment]
(please do not merge this PR as is, it won't get through to the application anyway)

**Comment by eu-genia** [comment]
Yes, thank you

**Comment by CarstenHoffmannMarburg** [comment]
I made 11 new pull requests with 10 records in each pull request:
No. 1 - #745 
No. 2 - #746 
No. 3 - #747 
No. 4 - #748 
No. 5 - #749 
No. 6 - #750 
No. 7 - #751 
No. 8 - #753
No. 9 - #754
No. 10 - #755
No. 11 - #752

**Pull request closed without merging**
